tin/apps/courses/models.py

```diff
@@ -35,6 +51,9 @@ class Course(models.Model):
 
     sort_assignments_by = models.CharField(max_length=30, choices=SORT_BY, default="due_date")
 
+    archived = models.BooleanField(default=False)
+    permission = models.CharField(max_length=30, choices=PERMISSIONS, default="r")
```

Comment:
If you do decide to go with the single-character choice proposal above, consider `max_length=1`.